pkg/operation/operation.go

```diff
@@ -115,6 +115,7 @@ func newOperation(
 		K8sGardenClient:        k8sGardenClient,
 		K8sGardenCoreInformers: k8sGardenCoreInformers,
 		ChartApplierGarden:     kubernetes.NewChartApplier(renderer, applier),
+		ChartsRootPath:         common.ChartPath,
```

Comment:
Why is this needed when we have the `common.ChartPath` constant?

Reply:
Because for tests, this won't work.